### PR TITLE
Mesh conversion function update (and other fixes)

### PR DESCRIPTION
**Pull request by jdent02**

### CODE REVIEW

File: translators/scene.py [hunk 1]
```diff
@@ -625,10 +625,11 @@ def __translate_render_settings(self):
                                   'max_path_length': 0,
                                   'max_photons_per_estimate': asr_scene_props.sppm_max_per_estimate,
                                   'path_tracing_max_path_length': asr_scene_props.sppm_pt_max_length,
-                                  'path_tracing_max_ray_intensity': asr_scene_props.sppm_pt_max_ray_intensity,
                                   'path_tracing_rr_min_path_length': asr_scene_props.sppm_pt_rr_start,
                                   'photon_tracing_max_path_length': asr_scene_props.sppm_photon_max_length,
                                   'photon_tracing_rr_min_path_length': asr_scene_props.sppm_photon_rr_start}
+            if asr_scene_props.sppm_pt_max_ray_intensity != 0.0:
```
Comment (dictoon) [inline translators/scene.py]: In principle 0 is a valid value for MRI. The way this is supposed to work is: if MRI is set, apply it; if it's not set, don't apply it; don't filter by value.

File: translators/mesh.py [hunk 2]
```diff
@@ -336,7 +336,7 @@ def __convert_mesh(self, me):
 
             uv_layer_pointer = active_uv.data[0].as_pointer()
 
-        asr.convert_bl_mesh(self.__mesh_object,
+        asr.convert_tessface_mesh(self.__mesh_object,
                             vertices_length,
```
Comment (dictoon) [inline translators/mesh.py]: Fix indentation.

File: translators/scene.py (hunk 1 above)
Comment (dictoon) [reply translators/scene.py]: In blenderseed for Blender 2.79, there's an `Unlimited` button next to the MRI input. Here we should check the status of that button, not the value that is set. Does that make sense?

**Comment by jdent02** [comment]
Addressed your comments

**Comment by dictoon** [comment]
Good to go, please rename commit "Updates for feedback" or merge it with others.

**Pull request merged**